system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -25,3 +25,43 @@ pub type SnowbridgeExporter = EthereumBlobExporter<
 	snowbridge_pallet_outbound_queue::Pallet<Runtime>,
 	snowbridge_core::AgentIdOf,
 >;
+
```

Comment:
@claravanstaden can you please move all your pallet inits and `parameter_types! {` to this file from libs.rs?